packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -959,11 +964,8 @@ The device model defines several read-only properties:
       "model":"VMware7,1",
       "osVersion":"10.0.18362",
       "serialNumber":"56 4d 4f 95 74 c5 d3 e7-fc 3a 57 9c c2 f8 5d ce",
-      "imei":null,
```

Comment:
Should you be removing `manufacturer`, `model`, `osVersion`, `serialNumber` like in the above cases?